src/libsystemd/sd-device/device-enumerator-private.h

```diff
@@ -5,6 +5,7 @@
 
 int device_enumerator_scan_devices(sd_device_enumerator *enumeartor);
 int device_enumerator_scan_subsystems(sd_device_enumerator *enumeartor);
```

Comment:
btw, "enumerartor" is a typo, here and above

Reply:
Ouch, Ouch! Will fix soon.